docs/en/03_Upgrading/index.md

```diff
@@ -1126,7 +1126,22 @@ All your assets should be loading properly now.
 This is a good point to commit your changes to your source control system before moving on to the next step.
 
 
-## Step 10 - Running your upgraded site for the first time {#step10}
+## Step 10 - Update database class references {#step10}
+
+If you've updated your class names to use namespaces you will need to reflect those changes in any existing database. For example, if you've renamed your `HomePage` class to `App\HomePage` then the database `ClassName` column needs to be updated to point to the `App\HomePage` class, otherwise the CMS will tell you that the page is obsolete.
```

Comment:
"any existing database" - ...fields?

Reply:
Can you also maybe add a note like "this also applies to any polymorphic relationships"